contracts/truefi/SushiTimelock.sol

```diff
@@ -125,7 +125,7 @@ contract SushiTimelock {
 
         // YYY: // solium-disable-next-line security/no-call-value
         // solhint-disable-next-line avoid-call-value, avoid-low-level-calls
-        (bool success, bytes memory returnData) = target.call.value(value)(callData);
+        (bool success, bytes memory returnData) = target.call{value:value}(callData);
```

Comment:
This is to fix the `yarn build` warning, right? Let's move this change to a separate PR since it's unrelated to LoanToken2s.

Also since this contract is copied verbatim from Sushiswap without unit testing, let's add a comment to show the diff, e.g.:
```
// YYY: (bool success, bytes memory returnData) = target.call.value(value)(callData);
```
This helps auditors determine what we've changed and what's original.

And finally, a style nit: `target.call{value: value}`.

Thanks!